client/Commands/Prompts/RollDicePrompt.tsx

```diff
@@ -58,14 +73,20 @@ export const ShowDiceRollPrompt = (
     <div className="prompt--with-submit-on-right">
       <div>
         {"Rolled: "}
-        {diceExpression}
-        {" -> "}
+        {originalExpression}
+        {" → "}
         <span
           dangerouslySetInnerHTML={{ __html: rollResult.FormattedString }}
         />
-        <input className="response" type="number" value={rollResult.Total} />
+        <input
```

Comment:
This should be auto-focused in order for the user to be able to quickly copy the result.

Reply:
Gotcha, I wasn't aware of that use case of quickly copying the value. I was confused about it being editable and wanted more visual contrast. I'll take out the `disabled` part for sure and rethink the design elements.